--- a/Configure
+++ b/Configure
@@ -1677,7 +1677,9 @@ $config{CFLAGS} = [ map { $_ eq '--ossl-
 
 unless ($disabled{afalgeng}) {
     $config{afalgeng}="";
-    if (grep { $_ eq 'afalgeng' } @{$target{enable}}) {
+    if ($target =~ m/openwrt$/) {
+        push @{$config{engdirs}}, "afalg";
+    } elsif (grep { $_ eq 'afalgeng' } @{$target{enable}}) {
         push @{$config{engdirs}}, "afalg";
     } else {
         disable('not-linux', 'afalgeng');
